packages/transfer-app/src/SentBalance.styles.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { Header, SubHeader } from '@substrate/ui-components';
```

Comment:
Could we move these to Shared.styles.tsx in ui-components? seems like a set of components we could reuse throughout the appa

Reply:
I kept a `margin-top: 0;` here, as I think it's really specific to this case